pages/index.tsx

```diff
@@ -60,22 +60,24 @@ export default function Home({ deviceType }: Props) {
       coordinatesAndEventType?.sw_lng,
     ]
   );
-  const { error, isLoading, mutate, isValidating } = useSWR<
-    DataLite | undefined
-  >(url, dataFetcher, {
-    onLoadingSlow: () => setSlowLoading(true),
-    revalidateOnFocus: false,
-    onSuccess: (data) => {
-      if (!data) return;
+  const { error, isLoading, isValidating } = useSWR<DataLite | undefined>(
+    url,
+    dataFetcher,
+    {
+      onLoadingSlow: () => setSlowLoading(true),
+      revalidateOnFocus: false,
+      onSuccess: (data) => {
+        if (!data) return;
 
-      const transformedData = data.results ? dataTransformerLite(data) : [];
-      setMarkerData(transformedData);
-    },
-  });
+        const transformedData = data.results ? dataTransformerLite(data) : [];
+        setMarkerData(transformedData);
+      },
+    }
+  );
 
   const { setDevice } = useMapActions();
   const [remainingTime, resetThrottling] = useIncrementalThrottling(
-    mutate,
+    () => setURL(areasURL + "?" + urlParams),
```

Comment:
`useCallback` ile wrap edersek performansı etkileyecek muhtemel yan etkilerden kurtulmuş oluruz. `useIncrementalThrottling` callback'i `useEffect` için dependency olarak tutuyor.